src/Compilers/VisualBasic/Portable/Symbols/SymbolExtensions.vb

```diff
@@ -436,7 +436,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         <Extension>
         Friend Function ContainingNonLambdaMember(member As Symbol) As Symbol
-            While member?.Kind = SymbolKind.Method AndAlso DirectCast(member, MethodSymbol).MethodKind = MethodKind.AnonymousFunction
+            While (member?.Kind = SymbolKind.Method).GetValueOrDefault() AndAlso DirectCast(member, MethodSymbol).MethodKind = MethodKind.AnonymousFunction
```

Comment:
>(member?.Kind = SymbolKind.Method).GetValueOrDefault() [](start = 18, length = 54)

This is a fix, we are going to dereference member after AndAlso. Prior to this PR, compiler would incorrectly apply GetValueOrDefault call itself.